### DISCUSSION TITLE
Filter the jobs by location on page load?

### DISCUSSION THREAD

**Discussion by osmgit** [bullhorn/career-portal#530] [Ideas]
Hi, 

We want to have different landing pages for listing jobs in different areas.

Is there any way to filter the jobs by location on page load?

I think this might be a similar question here: https://github.com/bullhorn/career-portal/discussions/433

Thanks!

**Comment by pbearne** [comment]
Hi 

If you are n WordPress, you can swap to the Matador Jobs plugin and this works out of the box as we save location as taxonomies 

Paul
